packages/api/src/modules/notify/notify.service.test.ts

```diff
@@ -0,0 +1,43 @@
+import { NotificationType } from "./@types/NotificationType";
+import { NotifyOutPipe } from "./@types/NotifyOutPipe";
+import { NotifyService } from "./notify.service";
+
+describe("NotifyService", () => {
+    describe("notify", () => {
+        let notifyService: NotifyService;
+
+        beforeEach(() => {
+            notifyService = new NotifyService();
+        });
+
+        it("should call outPipe notify", async () => {
+            const fakeOutPipe: NotifyOutPipe = {
+                accepts: [NotificationType.TEST_EMAIL],
+                notify: jest.fn(),
+            };
+
+            // @ts-expect-error outPipes is private attributes
+            notifyService.outPipes = [fakeOutPipe];
+
+            const expected = { email: "Fake email", templateId: 0 };
+
+            await notifyService.notify(NotificationType.TEST_EMAIL, expected);
+
+            expect(fakeOutPipe.notify).toHaveBeenCalledWith(NotificationType.TEST_EMAIL, expected);
+        });
+
+        it("should call only good outPipe", async () => {
```

Comment:
Plutôt `"should not call outPipe"` ?

Reply:
Je trouve que ce serait moins précis. Si on veut être plus clair peut-être "should only call outPipe of required notificationType"

Reply:
oui mais à ce moment là je ferai un test avec un outPipe dans le boucle, ici c'est `{}`